DAOS-14480 test: Enable ud in ci.

jgmoore-or opened the pull request:
### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

jgmoore-or committed:
DAOS-14480 test: Extend Python CI code to enable ucx+ud_x provider.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-14480 test: Enable ud provider in CI.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-14430 test: Enable ud in ci.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/1/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/1/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/1/display/redirect

jgmoore-or committed:
DAOS-14480 test: Enable ud in ci.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+dc_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-14460 test: Enable ud in ci.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/4/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/4/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/4/display/redirect

jgmoore-or committed:
DAOS-14480 test: Enable ud in ci.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/5/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/5/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/5/display/redirect

jgmoore-or committed:
DAOS-14480 test: Enable ud in ci.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/6/execution/node/784/log

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-13529/6/testReport/

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/6/execution/node/800/log

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-13529/7/testReport/

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/7/execution/node/748/log

jgmoore-or committed:
DAOS-14480 test: Enable ud in ci with latest 23.10 mofed.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-func-test-vm: true
Skip-unit-tests: true
Test-tag: pr daily_regression
Test-provider: ucx+ud_x
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/8/execution/node/708/log

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13529/8/display/redirect

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/9/execution/node/739/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/9/execution/node/749/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13529/9/execution/node/895/log
